src/plugins/saved_objects_management/public/management_section/object_view/components/not_found_errors.tsx

```diff
@@ -76,13 +76,13 @@ export const NotFoundErrors = ({ type }: NotFoundErrors) => {
       iconType="alert"
       color="danger"
     >
-      <div>{getMessage()}</div>
-      <div>
+      <p>{getMessage()}</p>
+      <p>
```

Comment:
`<div></div>` should be replaced with `<p></p>`, in order to indicate a paragraph of text